creator-node/src/ffprobe-exec.js

```diff
@@ -0,0 +1,35 @@
+const ffprobeStatic = require('ffprobe-static')
```

Comment:
@hareeshnagaraj do we want to delete the old ffprobe file?

Reply:
yes